ui/src/dashboards/actions/thunks.ts

```diff
@@ -229,6 +233,34 @@ export const getDashboards = () => async (
     )
 
     dispatch(setDashboards(RemoteDataState.Done, dashboards))
+
+    if (!dashboards.result.length) {
```

Comment:
is `result` guaranteed to be exist and to be a array-like? we often have undefined errors because of assumptions that variables exist, just want to confirm this is kosher.

Reply:
totally is by the normalize function above